code/modules/cargo/packs.dm

```diff
@@ -1230,16 +1230,23 @@
 
 /datum/supply_pack/science/robotics
 	name = "Robotics Assembly Crate"
-	desc = "The tools you need to replace those finicky humans with a loyal robot army! Contains three proximity sensors, two high-powered cells, six flashes, and an electrical toolbox. Requires Robotics access to open."
-	cost = 1000
+	desc = "The tools you need to replace those finicky humans with a loyal robot army! Requires Robotics access to open."
```

Comment:
>he just removes any mention of what it contains, unlike literally any other crate order in the game

```suggestion
	desc = "The tools you need to replace those finicky humans with a loyal robot army! Contains enough bot parts to make two medibots. Requires Robotics access to open."
```

Reply:
Also what other bot can you make with this? Don't all of them need robotic arms, at the least?

Reply:
it contains the unique things robo doesnt have and needs he can print the robo arms in his lab